docs/user/instrument-android-crashes-example.md

```diff
@@ -0,0 +1,171 @@
+building any time one of the Glean YAML files has been modified.  
+
+It is recommended that Glean be initialized as early in the application startup as possible, which is why it's good to use a
+custom `Application`, like the Glean Sample App `GleanApplication` found [here](../../samples/android/app/src/main/java/org/mozilla/samples/glean/GleanApplication.kt)
+
+Initializing Glean in the `Application.onCreate()` is ideal for this purpose.  Start by adding the import statement to allow
+the usage of the custom ping that was created, adding the following to the top of the file:
+
+```Kotlin
+import org.mozilla.gleancrashexample.GleanMetrics.Pings
+```
+
+Next, register the custom ping by calling `Glean.registerPings(Pings)` in the `onCreate()` function, preferably before calling
+`Glean.initialize()`.  The completed fuction should look something like this:
+
+```Kotlin
+override fun onCreate() {
+  super.onCreate()
+
+  // Register the application's custom pings.
+  Glean.registerPings(Pings)
+
+  // Initialize the Glean library
+  Glean.initialize(applicationContext)
+}
+
+```
+
+This completes the registration of the custom ping with the Glean SDK so that it knows about it and can manage the storage and
+other important details of it like sending it when `send()` is called.
+
+### Instrument The App To Record The Event
+
+In order to make the custom `Application` class handle uncaught exceptions, extend the class definition by adding
+`Thread.UncaughtExceptionHandler` as an inherited class like this:
+
+```Kotlin
+class MainActivity : AppCompatActivity(), Thread.UncaughtExceptionHandler {
+    ...
+}
+```
+
+As part of implementing the `Thread.UncaughtExceptionHandler` interface, the custom `Application` needs to implement the
+override of the `uncaughtException()` function.  An example of this override that records data and sends the ping could look
+something like this:
+
+```Kotlin
+override fun uncaughtException(thread: Thread, exception: Throwable) {
+    Crash.exception.record(
+        mapOf(
+            Crash.exceptionKeys.cause to exception.cause!!.toString(),
+            Crash.exceptionKeys.message to exception.message!!)
+    )
+    Pings.crash.send()
+}
+```
+
+This records data to the `Crash.exception` metric from the `metrics.yaml`.  The category of the metric is `crash` and the name
+is `exception` so it is accessed it by calling `record()` on the `Crash.exception` object.  The extra information for the
+`cause` and the `message` is set as well.  Finally, calling `Pings.crash.send()` forces the `crash` ping to be scheduled to be
+sent.
+
+The final step is to register the custom `Application` as the default uncaught exception handler by adding the following to the
+`onCreate()` function after `Glean.initialize(this)`:
+
+```Kotlin
+Thread.setDefaultUncaughtExceptionHandler(this)
+```
+
+### Next Steps
+
+This information didn't really get recorded by anything, as it would be rejected by the telemetry pipeline unless the
```

Comment:
What do you mean here? Can you please expand on what's missing?

Reply:
I added the same link to the `probe_scraper` docs as found in the "Adding Glean to your project" section, which should cover the "next steps" required.